.github/workflows/codeql.yml

```
@@ -126,6 +127,18 @@ jobs:
           # For more details on CodeQL's query packs, refer to: https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning#using-queries-in-ql-packs
           # queries: security-extended,security-and-quality
 
+
+      - name: CMake Workflow with preset 'minimal-build' for packaging
+        uses: lukka/run-cmake@v10
+        with:
+          workflowPreset: 'minimal-build'
+        env:
+          CMAKE_GENERATOR: ${{ matrix.generator }}
+          CC: ${{ matrix.compiler == 'clang' && steps.set-up-clang.outputs.clang || matrix.compiler == 'gcc' && steps.set-up-gcc.outputs.gcc || '' }}
```

actionlint (1.7.7)

property "set-up-clang" is not defined in object type {set-up-gcc: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)